esp32-hal/README.md

```diff
@@ -25,7 +25,7 @@ espup install
 # Windows does not require sourcing any file
 ```
 
-See the [Installation chapter of The Rust on ESP Book](https://esp-rs.github.io/book/installation/installation.html#installing-rust-for-espressif-socs) for more details.
+See the [Installation chapter of The Rust on ESP Book](https://esp-rs.github.io/book/installation/index.html#risc-v-and-xtensa-targets) for more details.
```

Comment:
```suggestion
See the [Installation chapter of The Rust on ESP Book](https://esp-rs.github.io/book/installation/index.html) for more details.
```
I'd suggest linking to the install chapter instead of the riscv and xtensa section

Reply:
Sure :+1:  I'll amend my commit (applying your suggestion will create another one).